### PR TITLE
Feature/ivp 1136 adding super flag to add form

### PR DESCRIPTION
**Pull request by csciubbach**
Adding super restricted flag to the add form, and adding it

### CODE REVIEW

**Review by SteveHicksCH — COMMENTED** [review]
Looking good

File: package.json [hunk 1]
```diff
@@ -26,9 +26,9 @@
   },
   "homepage": "https://github.com/companieshouse/restricted-word-web#readme",
   "dependencies": {
-    "axios": "~0.19.2",
+    "axios": "~0.21.1",
     "ch-node-session-handler": "git+ssh://git@github.com/companieshouse/node-session-handler.git#1.0.0",
-    "ch-structured-logging": "git+ssh://git@github.com/companieshouse/ch-structured-logging-node.git#master",
+    "ch-structured-logging": "git+ssh://git@github.com/companieshouse/ch-structured-logging-node.git#15a8112",
```
Comment (SteveHicksCH) [inline package.json:31]: Looking at other projects (e.g. https://github.com/companieshouse/strike-off-objections-web/blob/master/package.json) they use a version number not commit reference (which seems better)

    "@companieshouse/structured-logging-node": "^1.0.4",
    
    Same in https://github.com/companieshouse/dissolution-web/blob/master/package.json

**Review by SteveHicksCH — COMMENTED** [review]
Just the one comment

File: package.json (hunk 1 above)
Comment (csciubbach) [reply package.json:31]: Because it requires that commit reference and it doesn't have a version

**Review by SteveHicksCH — APPROVED** [review]
Approved but we need a tech debt story to upgrade this project for changes in the Company House Libraries that have taken place since this application was written

**Pull request merged**